### DISCUSSION TITLE
Does this has a control bar on the left just like the demo in novnc?

### DISCUSSION THREAD

**Discussion by imnkywf** [roerohan/react-vnc#52] [General]
Does this has a control bar on the left just like the demo in novnc?

**Comment by imnkywf** [comment]
Help pls!!!

**Comment by roerohan** [comment]
Nope, it does not have that control bar
This is a React library which is a wrapper over noVNC and not a demo application, so it just provides methods to perform actions in React.

**Reply by imnkywf** [reply]
OK，thank you！
